src/index.html

```diff
@@ -1,13 +1,324 @@
+<!-- #region head -->
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>MyBike</title>
+    <link rel="icon" href="./images/icon/favicon.svg">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <!-- #endregion -->
+  <body class="page__body">
+    <!-- #region header -->
+    <header class="header">
+      <div class="header__content">
+        <div class="top-bar">
+          <a href="#" class="top-bar__logo-link">
+            <img
+              src="images/mybiKE-white-logo.svg"
+              alt="MyBike logo"
+              class="top-bar__logo"
+            >
+          </a>
+          <div class="top-bar__icons">
+            <a href="tel:+123455555555" class="icon icon--phone"></a>
+            <a href="#menu" class="icon icon--menu"></a>
+          </div>
+        </div>
+        <div class="header__bottom">
+          <h1 class="header__title">Take the Streets</h1>
+        </div>
+      </div>
+    </header>
+    <!-- #endregion -->
+
+    <!-- #region aside -->
+    <aside class="page__menu menu" id="menu">
+      <div class="menu__content">
+        <div class="top-bar menu__top">
+          <a href="#" class="top-bar__logo-link">
+            <img
+              src="images/mybiKE-black-logo.svg"
+              alt="MyBike logo"
+              class="top-bar__logo"
+            >
+          </a>
+          <a href="#" class="icon icon--close"></a>
+        </div>
+        <div class="menu__bottom">
+          <nav class="nav menu__nav">
+            <ul class="nav__list">
+              <li class="nav__item">
+                <a class="nav__link" href="#home">Home</a>
+              </li>
```

Comment:
Use spaces between li tags in that way: 
![image](https://github.com/mate-academy/layout_landing-page/assets/63805271/191e4dca-6ae2-411b-946b-5bf8d543bbc0)